README.md

```diff
@@ -1,8 +1,9 @@
 # bca-gpa-2023
+
 An update version of [the BCA GPA browser extension](https://github.com/alebod21/bcagpa) that fixes bugs due to PowerSchool updates, and also adds a port to Firefox.
 
 Chrome Web Store Page: https://chrome.google.com/webstore/detail/bca-gpa/ofhfmclagjfikdfhjnogmcokilbikmpf
 
 Firefox Addon Page: https://addons.mozilla.org/en-US/firefox/addon/bca-gpa/
 
-![Screenshot of extension](./screenshot.png)
+![Screenshot of extension](./images/screenshot.png)
```

Comment:
Can you update all of the screenshots to be stored as [githubusercontent instead of actually in the repository](https://www.geeksforgeeks.org/how-to-add-images-on-readme-md-file-in-a-github-repository-from-the-local-system/#)?

Reply:
all images except screenshot.png have been added to the GitHub assets of this repository and have been properly referenced in the readme

Reply:
would you like me to do the same for screenshot.png or leave it?

Reply:
If possible yes change that one too

Reply:
Done